src/v0/destinations/snowpipe_streaming/transform.js

```diff
@@ -1 +1,28 @@
-module.exports = require('../snowflake/transform');
+const transform = require('../snowflake/transform');
+const { processWarehouseMessage } = require('../../../warehouse');
+
+const provider = 'snowpipe_streaming';
+
+function process(event) {
+  const whSchemaVersion = event.request.query.whSchemaVersion || 'v1';
+  const whIDResolve = event.request.query.whIDResolve === 'true' || false;
```

Comment:
make this default false? not sure if we're using this now.

Reply:
Yeah, it might not be required. Removing it.